module/VuFind/src/VuFind/Search/Results/PluginManager.php

```diff
@@ -74,6 +75,8 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
      * @var array
      */
     protected $factories = [
+        \VuFind\Search\Blender\Results::class
+            => \VuFind\Search\Solr\ResultsFactory::class,
```

Comment:
If we're sharing the Solr factory with Blender, should we rename the factory to something more generic to prevent confusion?

Reply:
Blender Results actually descends from Solr results (like other classes as well) so that it works like Solr.